provider/marathon.go

```diff
@@ -179,7 +179,15 @@ func (provider *Marathon) getLabel(application marathon.Application, label strin
 	return "", errors.New("Label not found:" + label)
 }
 
-func (provider *Marathon) getPort(task marathon.Task) string {
+func (provider *Marathon) getPort(task marathon.Task, applications []marathon.Application) string {
+	application, errApp := getApplication(task, applications)
+	if errApp != nil {
+		log.Errorf("Unable to get marathon application from task %s", task.AppID)
+		return ""
+	}
+	if label, err := provider.getLabel(application, "traefik.port"); err == nil {
+		return label
```

Comment:
In the case of Marathon, I think it would be better to use `traefik.port` as port index (like in Marathon health API https://mesosphere.github.io/marathon/docs/health-checks.html). In case of multiple nodes Mesos cluster, we have to use mapped ports by Marathon. OK with that?